src/Symfony/Component/ExpressionLanguage/ExpressionLanguage.php

```diff
@@ -62,7 +62,7 @@ public function evaluate(Expression|string $expression, array $values = []): mix
     /**
      * Parses an expression.
      */
-    public function parse(Expression|string $expression, array $names): ParsedExpression
+    public function parse(Expression|string $expression, array $names, int $flags = 0): ParsedExpression
```

Comment:
the methods in this class should also have the `@param int-mask-of<Parser::IGNORE_*> $flags` phpdoc

Reply:
Done